src/Maker/MakeController.php

```diff
@@ -80,14 +83,16 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
         $io->text('Next: Open your new controller class and add some pages!');
     }
 
-    public function configureDependencies(DependencyBuilder $dependencies)
+    public function configureDependencies(DependencyBuilder $dependencies, InputInterface $input = null)
     {
-        $dependencies->addClassDependency(
-            // we only need doctrine/annotations, which contains
-            // the recipe that loads annotation routes
-            Annotation::class,
-            'annotations'
-        );
+        if (null === $input) {
```

Comment:
I am not completely sure about this myself. If there is a better way to handle this, please feel free to edit or comment.